packages/create-app/build.mjs

```diff
@@ -0,0 +1,33 @@
+import { copyFileSync, existsSync, mkdirSync, rmSync } from 'node:fs'
+import * as url from 'node:url'
+import { resolve } from 'node:path'
+
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url))
+const __templateDir = resolve(__dirname, 'template')
+const __pagesDir = resolve(__templateDir, 'pages')
+
+const shouldCreatePagesDict = () => !existsSync(__pagesDir)
+
+// key: copy to (relative ./)
+// value: origin (relative ./template)
+const needCopyFiles = {
+  'slides.md': '../../../demo/starter/slides.md',
+  'pages/multiple-entries.md': '../../../demo/starter/pages/multiple-entries.md',
```

Comment:
These files don't exist while scaffolding the app with the create-app package; the create-app folder is effectively the top level in those cases. This is also a security issue as you are effectively trying to access unknown files in a random user's folder